seafile/SeafFileViewController.m

```diff
@@ -786,7 +803,11 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 
     if (tableView != self.tableView) {
         // For search results.
-        return [self getSeafFileCell:(SeafFile *)entry forTableView:tableView andIndexPath:indexPath];
+        if ([entry isKindOfClass:[SeafDir class]]) {
```

Comment:
For folders in search view, you should use the absolute path as the detail text(the same as file in getSeafFileCell). What happened if user click on the folder?